test_plugin/tests/integration_tests.rs

```diff
@@ -0,0 +1,44 @@
+use deno_cli::test_util::*;
+use std::process::Command;
+
+fn deno_cmd() -> Command {
+  assert!(deno_exe_path().exists());
+  Command::new(deno_exe_path())
+}
+
+#[cfg(debug_assertions)]
+const BUILD_VARIANT: &str = "debug";
+
+#[cfg(not(debug_assertions))]
+const BUILD_VARIANT: &str = "release";
+
+#[test]
+fn basic() {
+  let mut build_plugin_base = Command::new("cargo");
+  let mut build_plugin =
+    build_plugin_base.arg("build").arg("-p").arg("test_plugin");
```

Comment:
It's annoying that it's not built already when the integration tests run. I wish I knew some cargo expert who could advise us here.